__mocks__/path.js

```diff
@@ -2,18 +2,11 @@
 
 const path = jest.genMockFromModule('path');
 
-const resolve = (dir, filepath) => {
-	return filepath;
-};
+const sep = process.platform === 'win32' ? '\\' : '/';
```

Comment:
Does the "sep" property actually need to be mocked? I don't think the actual property implementation touches the file system, so you might be ok without a mock. It would mean less code to write/maintain, at least.